### ISSUE TITLE
Unable to equip loadout (1.2.6 from Chrome store)

### ISSUE THREAD

**Issue by ssoriche** [dasilva333/DIM#5]
When I click on a character to apply the loud out to, nothing happens and the error log contains:

```
loadouts.js:172 Uncaught TypeError: undefined is not a function
loadouts.js:172 (anonymous function)
loadouts.js:118 transfer
VM479:1 (anonymous function)
knockout.js:83 (anonymous function)
jquery.js:3 m.event.dispatch
jquery.js:3 r.handle
```

Also from the screenshot below, there appears to be text that is supposed to be over each of the character icons, but it blends in.

![screen shot 2015-03-29 at 9 18 00 pm](https://cloud.githubusercontent.com/assets/19838/6888933/2cd2e6b8-d659-11e4-8251-442bc1e57874.png)

**Comment by ssoriche** [comment]
Transfers have been corrected in 1.2.8, but the dark text remains.

**Comment by ssoriche** [comment]
Dark text has been placed below the emblem.
